package_control/automatic_upgrader.py

```diff
@@ -332,3 +333,30 @@ def disable_packages():
                     ''',
                     (package_name, version)
                 )
+
+    def upgrade_depenendencies(self):
+        """
+        Upgrades all dependencies that are not currently upgraded to the lastest version.
+        """
+
+        if not self.auto_upgrade:
+            return
+
+        dependency_list = self.installer.make_package_list(
+            [
+                'install',
+                'reinstall',
+                'downgrade',
+                'overwrite',
+                'none'
+            ],
+            ignore_packages=self.auto_upgrade_ignore,
+            get_dependencies=True
+        )
+
+        if USE_QUICK_PANEL_ITEM:
+            dependency_list = [info.trigger for info in dependency_list]
```

Comment:
Note: Here and in the original `upgrade_packages` method, the handling of `USE_QUICK_PANEL_ITEM` seems pointless because this is the automatic upgrade and it never presents this info in the UI. So handling of it here and in `PackageInstaller.make_package_list` could be removed to simplify things.

Reply:
Isn't it just to be able to interpret the result of `self.installer.make_package_list`, which returns a `List[QuickPanelItem]` if this flag is true?

Reply:
I've mentioned that the handling could be removed in **both places** because the result of `make_package_list` is never presented in the UI so it doesn't have to return results in "quick panel" format.